.github/workflows/javadoc.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
This line is not required, as it's the default.

Reply:
Agreed, I removed it. [link](https://github.com/ably/ably-java/pull/832/files/6a9746d2ab467f28d88e3d1bac496dd055a44b9c..8f7b0ece28742fd7c86dcf9ac383c933063a4c7f)